src/main.d

```diff
@@ -858,7 +862,11 @@ int main(string[] args)
 				auto currTime = MonoTime.currTime();
 				if (currTime - lastCheckTime > checkInterval) {
 					// monitor sync loop
-					log.vdebug("################################################## NEW LOOP ##################################################");
+					if (displaySyncOptions) {
+						log.log("################################################## NEW LOOP ##################################################");
+					} else {
+						log.vdebug("################################################## NEW LOOP ##################################################");
+					}
```

Comment:
Net sure whether D supports pointers to functions, but initializing `logfunction -> log.log` or `-> log.vdebug` and then not duplicating all the logging lines might be useful. Decluttering, maintainability?

Reply:
will have to look into this if an option - but at a high level, it is only these log lines that I have been using - nothing else. If initialising to something, it may mean then all verbose debugging now would come through, which is not the intention.

The use case here is - run without say '--verbose' but enable this option, and you can see at a quick snapshot what is going on, without any other noise.